decompress_tar.go

```diff
@@ -90,8 +91,16 @@ func untar(input io.Reader, dst, src string, dir bool) error {
 			return err
 		}
 
-		// Set the access and modification time
-		if err := os.Chtimes(path, hdr.AccessTime, hdr.ModTime); err != nil {
+		// Set the access and modification time if valid, otherwise default to current time
+		aTime := time.Now()
+		mTime := time.Now()
```

Comment:
These two lines could result in two different values, can we do `mTime := aTime` instead. `time.Time` is just a value, so it'll copy.